docs/2024/ci-scanner/updates/2024-06-20.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Week 3
+author: Rajul Jha
+---
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2024 Rajul Jha <rajuljha49gmail.com>
+-->
+
+# Week 3
+*(June 14, 2024 - June 20, 2024)*
+
+## Meeting 1
+*(June 19, 2024)*
+
+## Attendees
+* [Rajul Jha](https://github.com/rajuljha)
+* [Kaushlendra](https://github.com/Kaushl2208)
+* [Shaheem Azmal](https://github.com/shaheemazmalmmd)
+* Katharina
+
+
+## Discussions
+
+* Worked on adding nomos json output. But not able to access the theMatches object's licenseAndMatchPositions object. Stuck on this for nomos task. 
+* Bumped up spdx_tools library to latest version. Tested with both platforms (GH Actions and Gitlab).
+* Discussed an issue in the github workflow, not supporting multiple architectures.
+  * The GH Actions Runner does not support multi-architecture images.
+  * We discussed potential solutions for the same including trying to build the image in the GH Action itself or utilizing an emulator like qemu. 
+
+## Work Done
+
+* Upgraded the spdx_tools library [(#PR2762)](https://github.com/fossology/fossology/pull/2762) and did performance analysis for both versions.
+    * With Version `0.0.0a2`: 
+    ![Screenshot](/img/ci/Spdx_tools_Version_0.8.2.png)
+    
+    * With Version `0.8.2`:
+    ![Screenshot](/img/ci/Spdx_tools_Version_0.8.2.png)
```

Comment:
File for both images are same. Might wanna check?

Reply:
Corrected